config/settings/base.py

```diff
@@ -282,6 +284,53 @@
 SOCIALACCOUNT_ALLOW_REGISTRATION = env.bool("DJANGO_SOCIALACCOUNT_ALLOW_REGISTRATION", True)
 ACCOUNT_ALLOWED_EMAIL_DOMAINS = env.list("DJANGO_ACCOUNT_ALLOWED_EMAIL_DOMAINS", default=["opencraft.com"])
 
+# Jira configs
+JIRA_SERVER = env.str("DJANGO_JIRA_SERVER")
+JIRA_USERNAME = env.str("DJANGO_JIRA_USERNAME")
+JIRA_PASSWORD = env.str("DJANGO_JIRA_PASSWORD")
+JIRA_SPRINT_BOARD_PREFIX = env.str("DJANGO_SPRINT_BOARD_PREFIX", "Sprint - ")
+JIRA_CUSTOM_FIELDS = env.dict("DJANGO_JIRA_CUSTOM_FIELDS", default={
+    "sprint": 'customfield_10005',
+    "story_points": 'customfield_10002',
+    "reviewer_1": 'customfield_10200',
+    "reviewer_2": 'customfield_10201',
+})
+JIRA_BOT_USERNAME = env.str("DJANGO_JIRA_BOT_USERNAME", "crafty")
```

Comment:
Could you please add a bit more descriptive comments to the configuration? ([example](https://gitlab.com/opencraft/dev/jira-scripts/blob/168fa3ccf15b9b5fb59f0fe60d8e779bfdab98a4/com/opencraft/jira/planning/Config.groovy#L7-24))

For example, as someone who hasn't tried this out yet, it's unclear to me what the difference between `JIRA_USERNAME` and `JIRA_BOT_USERNAME` is, or where to get the data in `JIRA_CUSTOM_FIELDS`